.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish-gpr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 10
+          registry-url: https://npm.pkg.github.com/
+          scope: '@adrise'
```

Comment:
will this `scope` have effects? it seems mismatched with what scope it contains in package.json